src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -337,6 +337,47 @@
  * <a href="https://github.com/checkstyle/checkstyle/issues/4830">issue 4830</a>.
  * </p>
  * <p>
+ * Currently, filter supports the following xpath axes:
```

Comment:
Just FYI for others: https://www.saxonica.com/html/documentation/expressions/axissteps.html